jdaviz/configs/default/plugins/markers/markers.py

```diff
@@ -127,15 +178,19 @@ def _on_viewer_added(self, msg):
         self._create_viewer_callbacks(self.app.get_viewer_by_id(msg.viewer_id))
 
     def _recompute_mark_positions(self, viewer):
-        if self.table is None or self.table._qtable is None:
+        if self.table is None or len(self.table.items) == 0:
             return
         if 'world_ra' not in self.table.headers_avail:
             return
 
+        qtable = self.table.export_table()
+        if qtable is None:
+            return

```

Comment:
wasn't this covered in the old version above? `if ... self.table._qtable is None`?